tasks/release.rb

```diff
@@ -314,7 +314,7 @@ def major_or_security?
     end
 
     def rc?
-      @version =~ /rc/
+      @version.include?("rc")
```

Comment:
This feels like a great example of something that I'd argue is neither simpler nor more idiomatic, despite being unquestionably less computationally intensive. :confused: